lua/wire/gpulib.lua

```diff
@@ -83,13 +83,15 @@ if CLIENT then
 	end
 
 	-- Returns a render target from the cache pool and marks it as used
-	local function GetRT()
-
+	local function GetRT(resx,resy)
+		resx = resx or 512
+		resy = resy or 512
 		for i, RT in pairs( RenderTargetCache ) do
 			if not RT[1] then -- not used
-
 				local rendertarget = RT[2]
 				if rendertarget then
+					rendertarget = GetRenderTargetEx("WireGPU_RT_"..i.."_"..resx.."_"..resy, resx, resy, RT_SIZE_LITERAL, MATERIAL_RT_DEPTH_SEPARATE, bit.bor(2, 256), 0, IMAGE_FORMAT_BGRA8888)
+					RT[2] = rendertarget
```

Comment:
This will allow vRAM leaks

Reply:
how exactly?

Reply:
> how exactly?

It's creating new rendertargets and over time they will use up all of VRAM. There's no way to clean up rendertarget memory.

Reply:
And i cant reuse the rendertargets? Presumably because i cant change the resolution if i just GetRenderTargetEx with the same name but different size?

Reply:
I think you can reuse them if they have the same name, but I'm not sure if their size can change.